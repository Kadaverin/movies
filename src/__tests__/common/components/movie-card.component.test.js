import React from 'react';
import { shallow } from 'enzyme';
import MovieCard from '../../../common/components/movie-card/movie-card';

const requiredProps = {
  imgUrl: 'url',
  title: 'card',
  releaseDate: '2019-07-19',
  voteCount: 100,
  voteAverage: 9.2,
};

describe('common component', () => {
  describe('MovieCard', () => {
    test('mathes shapshot', () => {
      const card = shallow(<MovieCard {...requiredProps} />);

      expect(card).toMatchSnapshot();
    });
  });
});
