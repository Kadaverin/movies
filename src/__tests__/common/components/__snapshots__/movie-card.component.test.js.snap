// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common component MovieCard mathes shapshot 1`] = `
<WithStyles(ForwardRef(Card))
  className="makeStyles-card-1"
>
  <WithStyles(ForwardRef(CardActionArea))
    className="makeStyles-cardActionContent-2"
  >
    <div>
      <WithStyles(ForwardRef(CardMedia))
        alt="poster"
        component="img"
        height={300}
        image="url"
      />
      <WithStyles(ForwardRef(CardContent))>
        <WithStyles(ForwardRef(Typography))
          component="h2"
          gutterBottom={true}
          variant="h5"
        >
          card
        </WithStyles(ForwardRef(Typography))>
        <WithStyles(ForwardRef(Typography))
          color="textSecondary"
          component="p"
          variant="body2"
        >
          2019-07-19
        </WithStyles(ForwardRef(Typography))>
      </WithStyles(ForwardRef(CardContent))>
    </div>
    <WithStyles(ForwardRef(CardActions))
      disableSpacing={true}
    >
      <Raiting
        variant="body2"
        voteAverage={9.2}
        voteCount={100}
      />
    </WithStyles(ForwardRef(CardActions))>
  </WithStyles(ForwardRef(CardActionArea))>
</WithStyles(ForwardRef(Card))>
`;
